src/components/utils/onfidoApi.js

```diff
@@ -7,7 +7,8 @@ const formatError = ({response, status}, onError) =>
   onError({status, response: JSON.parse(response)})
 
 export const uploadDocument = (data, token, onSuccess, onError) => {
-  data['sdk_validations'] = {'detect_document': 'error', 'detect_glare': 'warn'}
+  const validations = {'detect_document': 'error', 'detect_glare': 'warn'}
+  data['sdk_validations'] = JSON.stringify(validations)
```

Comment:
Ricardo's comment on my PR: "better never to change objects in place unless it's made clear. a simple data= {...data, sdk_validations: ... will do the trick"